src/thread/Makefile

```diff
@@ -8,7 +8,7 @@ TESTFILES = kaldi-thread-test kaldi-task-sequence-test
 OBJFILES =  kaldi-thread.o kaldi-mutex.o kaldi-semaphore.o kaldi-barrier.o
 
 LIBNAME = kaldi-thread
-ADDLIBS = ../matrix/kaldi-matrix.a ../base/kaldi-base.a
+ADDLIBS = ../base/kaldi-base.a
```

Comment:
It does not seem to need the kaldi-matrix (and was confusing make -j8).
make depend would probably fix the confusion, but this seemed better solution